code/graph-algorithms/eulerian_path/eulerian.py

```diff
@@ -0,0 +1,118 @@
+# Python program to check if a given graph is Eulerian of not using eulerian path identification
+#Complexity : O(V+E)
```

Comment:
OpenGenus comment needed.

Reply:
My apologies but I didn't get u . Could u elaborate please ?

Reply:
Please add comment  /* Part of Cosmos by OpenGenus Foundation */ at the top of every code you contribute to this project  ! 👍